CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - `label:properties` as array needs at least one item
-- `name` in `label:classes` must be non-empty
+- `name` in `label:classes` must be non-empty or null
```

Comment:
I think this could be a bit more precise --

```suggestion
- `name` in `label:classes` must be non-empty for vector label items
- `name` in `label:classes` must be `null` for raster label items (#8)
```

Reply:
Sorry for the two part review / 🧇-ing, I was too focused on the json schema and missed the changelog entry on first review